### ISSUE TITLE
Review "failing" sample files for multiple failures, split into separate files.

### ISSUE THREAD

**Issue by beporter** [loadsys/loadsys_codesniffer#45]
Some of the sample files currently fail multiple sniffs. Our testing harness can catch multiple sniff failures per file, but it makes the tests relatively inflexible since the files aren't "atomic" compared to the sniff rules.

It's also possible for the test files to be failing tests they really shouldn't be, such as overlapping rules that cover the same "problem" with the code. In these cases, the ruleset needs to be modified to try to only include one relevant rule to cover the failure.

Bottom line is: The files in `snifftests/files` that contain first lines like this:

```php
<?php //~Standard.Section.Sniff.Rule, Another.Failing.Rule.Name
```

(The ones that define multiple failing rules.) Those need to be reviewed and split up where possible, or the ruleset modified to not cause overlapping failures. (The CakePHP sniffs are responsible for a lot of overlap against PSR-2, and typically the Cake sniffs lack the "autofix" capability the PSR-2 sniffs include.)
